card/mod/05_standard/spec/set/type/signup_spec.rb

```diff
@@ -175,9 +175,9 @@
     end
 
     it 'sends welcome email when account is activated' do
-      @signup.run_phase :approve do
+      #@signup.run_phase :approve do
```

Comment:
Incorrect indentation detected (column 6 instead of 8).<br>Missing space after #.